geometry_msgs/PoseStamped goal
float64 position_tolerance
float64 orientation_tolerance
float64 planning_time
string group
---
bool success
